### PR TITLE
Add link to Launchpad bug tracker

### PR DESCRIPTION
**Pull request by buggtb**
Bubble up the bug tracker to the readme to make it easier to find as its not the Github issue site

### CODE REVIEW

**Comment by mitechie** [comment]
Awesome ty. $$merge$$

File: README.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ Explore questions and conversations on our [Discourse forum](https://jaas.ai/doc
 
 Developers are welcome to read through our [contributing guidelines](CONTRIBUTING.md) to learn how to make code changes.
 
+File bugs at our project [Launchpad](https://bugs.launchpad.net/juju/+bugs) site
```
Comment (timClicks) [inline README.md]: Could you please deep link to https://bugs.launchpad.net/juju/+filebug on that line? The Launchpad UI can be pretty bewildering if you haven't seen it before. 

```markdown
[File bugs](https://bugs.launchpad.net/juju/+filebug) at our [Launchpad project](https://bugs.launchpad.net/juju/+bugs) 
```

**Comment by timClicks** [comment]
@buggtb Minor point Tom - could you please update the commit message to something like "Add link to Launchpad bug tracker" rather than "Update README.md". We're trying to keep the log as clean as possible.

File: README.md (hunk 1 above)
Comment (buggtb) [reply README.md]: Done

**Comment by timClicks** [comment]
Sorry to be such a pain for such a small (but very useful handy!)

**Comment by buggtb** [comment]
Should be good now @timClicks

**Comment by timClicks** [comment]
$$merge$$

**Pull request merged**